### PR TITLE
Fixed the supplier bug

### PR DESCRIPTION
**Pull request by Bayoslav**
The if statement which checks whether an instance comes from a Manager type class on line 355 of shuup/admin/utils/picotable.py  did NOT return a value, hence why None was appearing instead of suppliers.

### CODE REVIEW

File: shuup/admin/utils/picotable.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def check_different_types(self, value):
 
         if isinstance(value, Manager):
             value = ", ".join("%s" % x for x in value.all())
-
+            return value
```
Comment (chessbr) [inline shuup/admin/utils/picotable.py]: please, leave a blank line after the return

**Comment by chessbr** [comment]
Also, make sure to sign the CLA above! Thanks for your contribution. :tada:

File: shuup/admin/utils/picotable.py [hunk 2]
```diff
@@ -355,6 +355,7 @@ def check_different_types(self, value):
         if isinstance(value, Manager):
             value = ", ".join("%s" % x for x in value.all())
             return value
+
```
Comment (chessbr) [inline shuup/admin/utils/picotable.py]: Please, rebase your 2 commits into a single one with all the changes

**Comment by tulimaki** [comment]
`./shuup/admin/utils/picotable.py:358:1: W293 blank line contains whitespace`

You can run `flake8` and `isort` to get some code styling tips.

Also since this is pretty much oneliner you shouldn't need to have more than one commit. You should be able to squash your commits with something like `git rebase -i HEAD~2`.

**Comment by chessbr** [comment]
@Bayoslav commits are just messed :sweat_smile: I recommend you working on a separated branch when submitting PRs. In your `master` branch do the following:

1. Add the `shuup` as a remote:
```
git remote add shuup git@github.com:shuup/shuup.git
```

2. Reset the current changes to the `shuup/master` state and use `--soft` to keep the changes in the stage
```
git reset --soft shuup/master
```

3. Create a new branch (I'll call it `fix-picotable` here, you can change the name) and commit the changes there:
```
git checkout -b fix-picotable
git add shuup/admin/utils/picotable.py 
git commit -m "Admin: fix picotable return value AND WRITE MORE MESSAGE HERE!"
git push origin fix-picotable
```

Then close this pull request and open another one using the new `fix-picotable` branch.

**Comment by Bayoslav** [comment]
Hey I know I messed up the commits with git rebase, will fix it when I get home.

**Comment by Bayoslav** [comment]
Hey sorry for messing it up, was planning on fixing it when I get home, your instructions throw an error, specifically at the 
`git reset --soft shuup/master` 
command so I deleted an old fork and created a new fork and a new pull request - https://github.com/shuup/shuup/pull/1284

**Pull request closed without merging**